canonical/ubuntu.com
# feat: build '/ai/data-science' page

## Done

- build '/ai/data-science' page based on [copydoc](https://docs.google.com/document/d/1GdVmQD9EDf_ksN6r1qWTYSwvdynCzAE8wtKaEdtzDo8/) and [design](https://www.figma.com/design/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=2668-10093&t=0bU85zbqKyAeSLdR-0)
- build new modal based on [form doc](https://docs.google.com/document/d/1dv-6XFaFXywPxEvD1fiklsLgic015H54ocesjArzs5E/edit)
- Added `margin-bottom: 1.25rem;` to the class `iti`, so the margins align with the rest of the form items. Also removes custom margin-top from the invalid number message

## QA

- Go to https://ubuntu-com-14050.demos.haus/ai/data-science
- Check the content matches the [copydoc](https://docs.google.com/document/d/1GdVmQD9EDf_ksN6r1qWTYSwvdynCzAE8wtKaEdtzDo8/)
- Check it visually matches the [design](https://www.figma.com/design/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=2668-10093&t=0bU85zbqKyAeSLdR-0)
- Check on different screen sizes
- Check the page works in terms of accessibility 
- Check the modal form matches the doc and the payload submits correctly, check it is recieved in [marketo](https://engage-sj.marketo.com/?munchkinId=066-EOV-335#/mktform/5917/overview/details)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12613
